read-the-docs-2019-stats.rst

```diff
@@ -0,0 +1,109 @@
+.. post:: May 14, 2020
+   :tags: stats, year-in-review
+
+Read the Docs 2019 Stats
+========================
+
+2019 was another good year for `Read the Docs`_.
+We continue to have a team of 5 folks working on the project,
+and we've rolled out a number of new features for the year.
+
+Here are our stats for the past year,
+which we've published since 2013.
+This is part of our effort to be transparent in our organization,
+as well as our source code.
+
+.. note:: 
+
+	You can always see our stats for the last `30 days`_. 
+
+	Our posts from 2013_, 2014_, 2015_, 2016_, 2017_, and 2018_ are also available.
```

Comment:
Shouldn't we list 2019 here as well?

Reply:
This post is 2019.

Reply:
Ha, I understood that you can click on those links and see _all the posts from the year_, but it's a link to the post with the stats.